job_executor/model/metadata.py

```diff
@@ -205,13 +208,40 @@ def dict(self, **kwargs) -> dict:  # pylint: disable=unused-argument
             dict_representation["keyType"] = self.key_type.dict(by_alias=True)
         return dict_representation
 
-    def patch(self, other: 'Variable'):
+    def patch(self, other: 'Variable') -> 'Variable':
         patched = {}
+        only_patch_description = False
+
         if other is None:
             raise PatchingError(
                 'Can not delete Variable'
             )
-        if (
+
+        if self.variable_role == "Identifier":
+            # Centralized variable definition was used,
+            # don't patch the one that is in the datastore.
+            # The definition might have changed and we don't want to update it
+            # - one needs to use CHANGE operation for that.
+            return self
+        if self.variable_role == "Measure" and self.key_type is not None:
+            # Centralized variable definition was used,
+            # it is safe to only patch name and description fields.
+            new_name = other.name
```

Comment:
the patchable field should be "label" 👀 ("name" in the SSB model translates to "label" in the SIKT model. The name field in the SIKT model is "shortName" in the SSB model which should always match the name of the variable itself. (The user will never be able to mess this up though, as it is auto inferred from filename in the whole system)

Reply:
Good catch. I fix